Core/clim-core/commands.lisp

```diff
@@ -49,7 +49,7 @@
   ((menu-name :reader command-menu-item-name :initarg :menu-name)
    (type :initarg :type :reader command-menu-item-type)
    (value :initarg :value :reader command-menu-item-value)
-   (documentation :initarg :documentation)
+   (documentation :initarg :documentation :initform nil)
```

Comment:
please write a more descriptive commit message

"Step one in fixing command menu problems" doesn't describe the change. You could write (for instance):

```
presentations: introduce a new presentation type command-menu-item

- add a new presentation type
- add a translator from the command-menu-item to command
- use the new presentation-type when displaying command table menu
- [list other changes]

These change is necessary to fix a bug mentioned in #955, that is
[put a brief description of the problem]. Closes #955.
```

It looks to me that all three commits do one atomic change (i.e we can't just take
the first commit and use it, because other changes does not follow), in that case
please squash all three commits into one and take all changes into account in
description. Atomic commits are important in case when we want to bisect changes
in order to find a regression.